src/model/AirLoopHVACUnitarySystem.hpp

```diff
@@ -276,71 +286,93 @@ namespace model {
 
     void resetSupplementalHeatingCoil();
 
-    OS_DEPRECATED(3, 7, 0) bool setSupplyAirFlowRateMethodDuringCoolingOperation(const std::string& supplyAirFlowRateMethodDuringCoolingOperation);
-    OS_DEPRECATED(3, 7, 0) void resetSupplyAirFlowRateMethodDuringCoolingOperation();
+    // During Cooling
 
+    /** Sets the field and switches "Supply Air Flow Rate Method During Cooling Operation" (coolingSAFMethod) to "SupplyAirFlowRate" */
     bool setSupplyAirFlowRateDuringCoolingOperation(double supplyAirFlowRateDuringCoolingOperation);
-    OS_DEPRECATED(3, 7, 0) void resetSupplyAirFlowRateDuringCoolingOperation();
+    /** Sets the field and switches the coolingSAFMethod to "SupplyAirFlowRate" */
```

Comment:
@kbenne should be return false if the unitary doesn't have a cooling coil set already?